### PR TITLE
Add to Multiply

### PR DESCRIPTION
**Pull request by JohnRoesler**

### CODE REVIEW

**Comment by antichris** [comment]
Why?

File: README.md [hunk 1]
```diff
@@ -113,8 +113,8 @@ var _ = Service("calc", func() {
 ```
 
 This file contains the design for a `calc` service which accepts HTTP GET
-requests to `/add/{a}/{b}` where `{a}` and `{b}` are placeholders for integer
-values. The API returns the sum of `a` and `b` in the HTTP response body.
+requests to `/multiply/{a}/{b}` where `{a}` and `{b}` are placeholders for integer
+values. The API returns result of `a` multiplied by `b` in the HTTP response body.
```
Comment (raphael) [inline README.md]: missing "the" (The API returns the result of...)

File: README.md [hunk 2]
```diff
@@ -183,15 +183,15 @@ is intended to generate just that: an *example*, in particular it is not
...
-`calc.go` and change the code of the `add` function as follows:
+request so all we have to do is to perform the actual multiplication. Edit the file
+`calc.go` and change the code of the `multiply` function as follows:
 
 ```go
-// Add returns the sum of attributes a and b of p.
-func (s *calcsrvc) Add(ctx context.Context, p *calc.AddPayload) (res int, err error) {
+// Multiply returns the multiplied value of attributes a and b of p.
+func (s *calcsrvc) Multiply(ctx context.Context, p *calc.MultiplyPayload) (res int, err error) {
         return p.A + p.B, nil
```
Comment (raphael) [inline README.md]: Please change implementation to `p.A * p.B`

**Review by raphael — CHANGES_REQUESTED** [review]
This is great, just two small changes needed

File: README.md (hunk 1 above)
Comment (antichris) [reply README.md]: Isn't it "the product"?

Comment (raphael) [reply README.md]: Yes that would be even better "The API returns the product of ..."

Comment (JohnRoesler) [inline README.md]:
```suggestion
values. The API returns the product of `a` multiplied by `b` in the HTTP response body.
```

File: README.md (hunk 2 above)
Comment (JohnRoesler) [inline README.md]:
```suggestion
        return p.A * p.B, nil
```

File: README.md (hunk 1 above)
Comment (JohnRoesler) [reply README.md]: done 👍

File: README.md (hunk 2 above)
Comment (JohnRoesler) [reply README.md]: done 👍

**Review by raphael — APPROVED** [review]

**Pull request merged**